plugins/woocommerce-blocks/docs/block-references/block-references.md

```diff
@@ -578,7 +578,7 @@ Payment options for your store.
 
 ## Pickup Method - woocommerce/checkout-pickup-options-block
 
-Shows local pickup options.
+Shows local pickup locations.
```

Comment:
There are more such changes in this file. I don't think these are directly related to removing the Beta label. Are these changes intentional?

Reply:
Changes are based on an automatic script [`build:docs:block-references`](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-blocks/package.json#L56). So it gets the newest data from blocks (title, descriptions, attributes etc.).

In order to update Product Collection name in Blocks Reference file, we also get all the other updates for free.

Reply:
Aha! Understood. I will perform smoke testing again and then approve the PR. Thanks for the explanation 🙌🏻